src/page-section/builder/renderer/Renderer.tsx

```diff
@@ -79,8 +85,10 @@ export const Renderer: React.FC<{
         </AspectRatio>
       </div>
       <div className="flex justify-center mt-4">
-        <Button onClick={downloadPNG}>Download</Button>
+        <Button onClick={downloadPNG} disabled={isLoading}>
+          {isLoading ? "Loading..." : "Download"}
```

Comment:
Show loader animation with the help of image and tailwind animation instead of Loading...